pkg/idl/doc.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+The IDL package describes comment directives that may be applied to API types.
+*/
+package idl
+
+// ListType annotates a list to further describe how it works.
+// Currently, it can have 3 possible values: "atomic", "map", or "set". Note
+// that there is no default, and eventually the generation step should
+// fail if a list is found that doesn't have that tag.
+//
+// Atomic
```

Comment:
```+listType=atomic```

And same for others: write exactly the line users will use, otherwise it is confusing to have capitals here, I think

Reply:
Yeah, this is a section header, so I think it has to start with a capital letter and finish with no points if I want to make this a title. I could definitely specify the actual full tag in the section thouhg